experiments/download_ssl4eo.py

```diff
@@ -576,21 +482,22 @@ def worker(idx: int) -> None:
         # add to existing checked locations
         with open(ext_path, "a") as f:
             writer = csv.writer(f)
-            if patches_s2c is not None:
+            if patches is not None:
                 success = 1
             else:
                 success = 0
-            data = [idx, center_coord[0], center_coord[1], success]
+            data = [idx, *center_coord, success]
             writer.writerow(data)
 
         return
 
     # set indices
     if args.indices_range is not None:
```

Comment:
I don't think it's possible for this to be None

Reply:
if args.indices_range is not specified, the script will just match the whole sampled coords

Reply:
But indices_range has a default value, it can't be set to None, right?